site/_apps/zuix-hn.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 2
+title: 'Zuix HN'
+github-title: 'genielabs/zuix-hn'
+library: 'ZUIX'
+libraries: 
+    - name: 'ZUIX'
+module-bundling: 'zuix-bundler'
+service-worker: 'Application Shell'
+perfomance-patterns: 'Lazy loaded components with caching'
+server-side-rendering: 'None'
+api: 'Hacker News Firebase API'
+hosting: 'Firebase'
+authors: 
+    - name: 'Gene'
+lighthouse: '91/100'
+interactive-em: '3.8s'
+interactive-faster-3g: '2.6s'
```

Comment:
Hmm if I'm not mistaken these are the `First View` times from WPT directly correct? Really sorry for the confusion, but what I meant is that with each Webpagetest, we make sure to generate Lighthouse results and then from there find the First Interactive times. I can't find associated LH results with your WPT tests so I'm assuming you didn't get it from there.

![image](https://cloud.githubusercontent.com/assets/12476932/26809349/02f915e8-4a32-11e7-87cd-bd96c51500c5.png)

![image](https://cloud.githubusercontent.com/assets/12476932/26809377/50d8c588-4a32-11e7-889e-09c2d15b1af5.png)

Please let me know if this makes sense. Since this seems ambigious, I really need to flesh out `CONTRIBUTING.md` and even include screenshots if necessary so I apologize if it wasn't clear enough.